### ISSUE TITLE
🐞 Display Conditions settings reset to "All" for location after saving

### ISSUE THREAD

**Issue by rblackfox** [elementor/elementor#26376]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

We are having issues with page templates & popups not saving display conditions after settings were selected and saved. 
We will select a page to apply to some location, we then save, and reopening the conditions the location changes back to "All" and does not display the correct location condition.

However on the fontend, the template does actually apply to the location selected, it's just in the conditional settings where it revert back to "All" for location.

Here is a video of the bug issue in action:
(https://drive.google.com/file/d/1s13SyO4oWgR6wCLU0OvX9MC7xRuQH_hw/view?usp=sharing)

### Steps to reproduce

1. Open disolay conditions and set a page  or popup to attach to a tag, category, wehatever
2. Select the "location" for that condition
3. Hit save
4. Open back up display conditions and that location has changed back to "All" and no longer shows the specific location chosen to display the content

### Expected behavior

1. Open display conditions select a display location, hit save
2. Refresh the page and reopen display conditions, the location shows some location the content it applied to other than "all"

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: mariadb.org binary distribution v10.6.17
	PHP Version: 8.2.18
	PHP Memory Limit: 56000M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.2
	Site URL:
	Home URL: 
	WP Multisite: Yes
	Max Upload Size: 15 MB
	Memory limit: 8192M
	Max Memory limit: 56000M
	Permalink Structure: /%category%/%postname%/
	Language: en_US
	Timezone: America/Denver
	Debug Mode: Inactive

== Theme ==
	Name: Child Hello
	Version: 1.0.1
	Author: 
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.0.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36

== Active Plugins ==
	Ahrefs SEO
		Version: 0.10.2
		Author: Ahrefs

	Bulk Updater for WooCommerce Subscriptions
		Version: 2.2.0
		Author: AspiringPlugins

	CheckoutWC
		Version: 9.0.35
		Author: Objectiv

	Coinbase Commerce
		Version: 1.3
		Author: Coinbase Commerce

	Custom Order Numbers for WooCommerce
		Version: 1.7.1
		Author: Tyche Softwares

	Discontinued Products for WooCommerce
		Version: 1.2.5
		Author: Kestrel

	Discount Rules Core
		Version: 2.6.3
		Author: Flycart

	Elementor
		Version: 3.21.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.21.1
		Author: Elementor.com

	Enhancer for WooCommerce Subscriptions
		Version: 4.4.0
		Author: Flintop

	Google Analytics for WooCommerce
		Version: 2.0.6
		Author: WooCommerce

	GTranslate
		Version: 3.0.6
		Author: Translate AI Multilingual Solutions

	HubSpot All-In-One Marketing - Forms, Popups, Live Chat
		Version: 11.1.6
		Author: HubSpot

	Infinite Scroll Elementor
		Version: 2.4.1
		Author: Joy Chetry

	Katalys
		Version: 1.1.19
		Author: Katalys

	Klaviyo
		Version: 3.3.5
		Author: Klaviyo, Inc.

	Metorik Helper
		Version: 1.7.1
		Author: Metorik

	NitroPack
		Version: 1.14.0
		Author: NitroPack Inc.

	Perfmatters
		Version: 2.1.8
		Author: forgemedia

	PixelYourSite PRO
		Version: 10.1.2.1
		Author: PixelYourSite

	Rank Math SEO PRO
		Version: 3.0.56
		Author: Rank Math

	Rank Math SEO with AI Best SEO Tools
		Version: 1.0.217
		Author: Rank Math

	RevOffers Advertiser Integration
		Version: 1.5.3
		Author: RevOffers

	Ultimate Addons for Elementor
		Version: 1.36.31
		Author: Brainstorm Force

	User Role Editor
		Version: 4.64.2
		Author: Vladimir Garagulya

	Variation Swatches for WooCommerce
		Version: 2.0.31
		Author: Emran Ahmed

	Web Accessibility by accessiBe
		Version: 1.17
		Author: accessiBe

	Woo All Products For Subscriptions
		Version: 5.0.3
		Author: Woo

	WooCommerce - ShipStation Integration
		Version: 4.4.2
		Author: WooCommerce

	WooCommerce.com Update Manager
		Version: 1.0.2
		Author: Automattic

	WooCommerce Advanced Notifications
		Version: 1.4.6
		Author: WooCommerce

	WooCommerce Authorize.Net Gateway
		Version: 3.10.2
		Author: SkyVerge

	WooCommerce Shipment Tracking
		Version: 2.4.7
		Author: WooCommerce

	WooCommerce Smart Coupons
		Version: 8.22.0
		Author: StoreApps

	WooCommerce USPS Shipping
		Version: 4.8.3
		Author: WooCommerce

	WooCommerce Zapier
		Version: 2.10.0
		Author: OM4 Software

	Woo Discount Rules PRO 2.0
		Version: 2.6.1
		Author: Flycart

	Woo Subscriptions
		Version: 6.2.0
		Author: WooCommerce

	WP Engine GeoTarget
		Version: 1.2.8
		Author: WP Engine

	WPForms
		Version: 1.8.8.3
		Author: WPForms

	WP Mail SMTP
		Version: 4.0.1
		Author: WP Mail SMTP

	WP Search with Algolia
		Version: 2.8.1
		Author: WebDevStudios

	WP Search with Algolia Pro
		Version: 1.4.0
		Author: Pluginize

	YITH Pre-Order for WooCommerce
		Version: 2.23.0
		Author: YITH

	YITH WooCommerce Wishlist Premium
		Version: 3.32.0
		Author: YITH

	Yotpo Social Reviews for Woocommerce
		Version: 1.7.8
		Author: Yotpo


== Network Plugins ==
	Discount Rules Core
		Version: 2.6.3
		Author: Flycart

	Object Cache Pro
		Version: 1.20.2
		Author: Rhubarb Group

	WooCommerce
		Version: 8.8.2
		Author: Automattic

	Woo Discount Rules PRO 2.0
		Version: 2.6.1
		Author: Flycart

	Wordfence Security
		Version: 7.11.5
		Author: Wordfence


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha: Active
	recaptcha_v3: Active
	woocommerce: Active


== Elementor Experiments ==
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Nested Elements Performance: Inactive by default
	Optimized Control Loading: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Inactive by default
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive
	Build with AI: Active by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Display Conditions: Inactive by default
	Form Submissions: Active
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Infinite Scroll Elementor: Compatibility not specified
	Ultimate Addons for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by rblackfox** [comment]
Any followup on this?

**Comment by nicholaszein** [comment]
Hello there @rblackfox!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.23.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
